source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -53,35 +53,31 @@ See the table below to see a description of each format:
    * - **Extended**
      - | Also known as the *canonical* format, this JSON representation avoids loss of BSON type information.
        | This format prioritizes type preservation at the loss of human-readability and interoperability with older formats.
-       |
-       | For more information on this format, see the server manual page on :manual:`MongoDB Extended JSON </reference/mongodb-extended-json/>`.
-
+       
    * - **Relaxed Mode**
      - | JSON representation that describes BSON documents with some type information loss.
        | This format prioritizes human-readability and interoperability at the loss of certain type information.
-       |
-       | For more information on this format, see the server manual page on :manual:`MongoDB Extended JSON </reference/mongodb-extended-json/>`.
 
    * - **Shell**
      - | JSON representation that matches the syntax used in the MongoDB shell.
        | This format prioritizes for compatibility with the MongoDB shell which often uses JavaScript functions to represent types.
-       |
-       | For more information on this format, see the server manual page on :manual:`Data Types in the mongo shell </core/shell-types/>`.
 
 .. _extended_json_example_section:
 
 .. note::
 
    The ``$uuid`` Extended JSON type is parsed from a String to a
-   :java-docs:`BsonBinary </apidocs/bson/org/bson/BsonBinary.html>`
-   object of binary subtype 4. For more information about ``$uuid`` field
-   parsing, see the
-   :spec:`special rules for parsing $uuid fields </extended-json.rst#special-rules-for-parsing-uuid-fields>`
-   section in the extended JSON specification.
-   
+   ``BsonBinary`` object of binary subtype 4. 
```

Comment:
```suggestion
   ``BsonBinary`` object of binary subtype 4. 
or more information about ``$uuid`` field   parsing, see the   :spec:`special rules for parsing $uuid fields &lt;/extended-json.rst#special-rules-for-parsing-uuid-fields&gt;`
```

Suggestion: I do not believe this :spec: link should be extracted from this note. I believe it only makes sense in the context of this note, and I believe readers not concerned with UUID parsing will be confused by seeing it in the end of section external resources.